src/components/presentation/NavBar.module.css

```diff
@@ -0,0 +1,19 @@
+.nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-end;
+  padding-bottom: 0.5rem;
+  gap: 1rem;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+  gap: 1rem;
+}
+
+button.buttonMargin {
```

Comment:
I totally agree, but there is something weird with the css specificity with the design system buttons, so it would be overridden by something internal (i think) if i left it out. The alternative was to use !important, or of course find the root of the problem in the design system 😅

Reply:
ah, then keep it as it is! Better than !important. 🙈